### PR TITLE
clients(devtools): expose registerLocaleData to worker

### PR DESCRIPTION
**Pull request by paulirish**

### CODE REVIEW

**Comment by paulirish** [comment]
> What do you think about removing registerLocaleData from the lighthouse module and calling it on i18n in here?

great suggestion. i'll do that.

File: clients/devtools-entry.js [hunk 1]
```diff
@@ -8,6 +8,7 @@
 const lighthouse = require('../lighthouse-core/index.js');
 const RawProtocol = require('../lighthouse-core/gather/connections/raw.js');
 const log = require('lighthouse-logger');
+const {registerLocaleData} = require('../lighthouse-core/lib/i18n/i18n.js');
```
[the remark this answers is not in the document]
Comment (paulirish) [reply clients/devtools-entry.js]: good call. done.

File: lighthouse-core/test/lib/i18n/i18n-test.js [hunk 2]
```diff
@@ -114,6 +114,16 @@ describe('i18n', () => {
...
 
   describe('#registerLocaleData', () => {
+    const moduleLocales = require('../../../lib/i18n/locales.js');
+    const clonedLocales = JSON.parse(JSON.stringify(moduleLocales));
+
+    // Restore locales to original state
+    afterEach(() => {
+      // eslint-disable-next-line no-unused-vars
+      let replacedLocales = require('../../../lib/i18n/locales.js');
+      replacedLocales = clonedLocales;
```
[the remark this answers is not in the document]
Comment (paulirish) [reply lighthouse-core/test/lib/i18n/i18n-test.js]: quite true. addressed in https://github.com/GoogleChrome/lighthouse/pull/9645/commits/cd9f8fda167e383c59e80745431fb893dac3a3f8

File: clients/devtools-entry.js [hunk 3]
```diff
@@ -63,4 +66,8 @@ if (typeof self !== 'undefined') {
   self.runLighthouseInWorker = runLighthouseInWorker;
   // @ts-ignore
   self.listenForStatus = listenForStatus;
+  // @ts-ignore
+  self.registerLocaleData = registerLocaleData;
+  // @ts-ignore
+  self.lookupLocale = lookupLocale;
```
Comment (brendankenny) [inline clients/devtools-entry.js]: @exterkamp FYI

**Pull request merged**